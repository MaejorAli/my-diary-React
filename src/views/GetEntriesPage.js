import React from 'react';
import GetEntries from '../containers/GetEntries';


export default (props) => (
  <div>
    <GetEntries {...props} />
  </div>
);
