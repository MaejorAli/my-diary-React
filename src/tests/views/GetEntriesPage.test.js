import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import GetEntriesPage from '../../views/GetEntriesPage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<GetEntriesPage />);
});


describe('Get Entries Page UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
