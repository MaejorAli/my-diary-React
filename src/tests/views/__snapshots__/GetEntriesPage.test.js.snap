// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get Entries Page UI render features view should render as expected 1`] = `
<div>
  <Connect(GetEntries) />
</div>
`;
